ad7768-1fmcz/ad77681.c

```diff
@@ -237,6 +237,35 @@ uint8_t ad77681_get_rx_buf_len(struct ad77681_dev *dev)
 	return rx_buf_len;
 }
 
+/**
+ * Helper function to get the number of SPI 16bit frames for INTERRUPT ADC DATA READ
+ * @param dev - The device structure.
+ * @return frame_16bit - the number of 16 bit SPI frames
+ */
+uint8_t ad77681_get_frame_16bit(struct ad77681_dev *dev)
```

Comment:
Add a description in the commit message, why this is needed and where

Reply:
Done adding description on the commit